tasks/pkg-suse.yml

```diff
@@ -1,16 +1,16 @@
 ---
-- block: # Work around due to SNI check for SLES11
```

Comment:
Have you tested this change doesn't break SLES11?

Reply:
I will test it tonight.

Reply:
@bhundven did you have a chance to run your test(s)?

Reply:
I have not. I don't have a suse setup to test on. Do you have any suggestions for testing this?

Reply:
If you split the PR in two, we can merge everything else now and wait for this part until someone can test it.

Reply:
@bhundven any feedback on Albert's last comment? Let us know if you don't have time to split this PR and we can take care of the Debian part for now...